template.yaml

```diff
@@ -722,6 +725,21 @@ objects:
           sslProfile: service_tls
           saslMechanisms: ANONYMOUS
       }  
+      
```

Comment:
I'd like to understand this configuration. I get this piece, but where is the logic that says to duplicate messages?

Reply:
The logic to multicast messages is actually not here but in line 715 of the template (in my draft PR): 
`address {
          prefix: broadcast
          distribution: multicast
      }
` 
What this tells the router is, for all addresses that begin with broadcast semantics, for instance if I had a queue named: broadcast.inventory.items, that those messages would get broadcast to all message listeners on Interconnects link graph. What I am trying to do with the connector configuration: 
` connector {
         name: remote-mesh-router
         role: inter-router
         host: ${REMOTE_ROUTER_HOST} #this should be the external ingress endpoint for the other router for instance: amq-interconnect-datacentre-b.apps.cluster-austin-5274.austin-5274.openshiftworkshop.com
         port: 443 #amqps
         sslProfile: inter_router_tls
         saslMechanisms: ANONYMOUS
         #cost: 100 #TODO mcostell we plan to multicast messages as a result, we likely want to weight each node on the graph to the same cost, adding this here to check that assumption  
         maxSessions: 10000 #FIXME mcostell ensure this makes sense. This is the maximum number of inflight sessions that can be made on the connection 
         linkCapacity: 10000 #FIXME mcostell ensure this makes sense. This is the number of inflight messages that can be in flight per link 
         messageLoggingComponents: all #FIXME mcostell this likely does not make sense in most contexts, leaving here to discuss the logging needs with the group  
         idleTimeoutSeconds: 3 #FIXME mcostell the default timeout is 16 seconds which would be far too high for our use case; however, 3 seconds might be too long 
         stripAnnotations: out #TODO it is likely dispatch router annotations should be stripped as it goes to the next router, check this assumption 
      }`
Is to hook up to the remote router as a peer, and with some attention to the effect that it might have on the router altogether, i.e. I want to constrain my inter-router behaviour to some degree as I want to have enough pipe to be able to take messages via normal means, which is why I've taken some initial first stabs at things like maxSessions or linkCapacity